### ISSUE TITLE
[Suggestion] Snowlogging and Gravity

### ISSUE THREAD

**Issue by seriousfreezing** [StartsMercury/visual-snowy-leaves#3]
I know it might be out of scope, but it could be configurable or another mod;
But the snow in the Bedrock edition has different behavior like Gravity, its drop and others.

**Comment by StartsMercury** [comment]
> I know it might be out of scope, but it could be configurable or another mod;
> But the snow in the Bedrock edition has different behavior like Gravity, its drop and others.

I find it great, getting suggestions, though, yes, it may be out of scope. I could make a separate mod, but I'll do some research because surely someone alread did this...or not. Make snow layer fall like sand instead of insta-break right? I think there are mods for layer stacking when snowing, I think. Hmm

**Comment by seriousfreezing** [comment]
There is no mod (as far as I know for 1.21 either) that brings most of the snow features from the Bedrock edition.

**Comment by seriousfreezing** [comment]
I think there are many more details that need to be changed, so I'd recommend looking at the [wiki page](https://minecraft.wiki/w/Snow) and seeing what might be possible. 

And yes many of the features would be server-side only, which is totally different from your mods already made.

**Comment by EGOIST1372** [comment]
@seriousfreezing @StartsMercury 
Hi
there is a mod called [snow real magic](https://www.curseforge.com/minecraft/mc-mods/snow-real-magic-fabric) that do all of those and more

**Comment by seriousfreezing** [comment]
Some time after posting this, I discovered this mod. However, it is still not suitable for my use because there is no version for 1.21.8.

**Comment by EGOIST1372** [comment]
yeah that's why I made issues on similar mods to see which dev willing to implement such features
there is another accumulating snow mod called [stiched snow](https://www.curseforge.com/minecraft/mc-mods/stitchedsnow) that is updated up to 1.21.5
